[docs] Updates docs about FAQ control-plane-manager (#13045)
deckhouse/deckhouse: modules/040-control-plane-manager/docs/FAQ.md

1. Restart the master node.

### What to do if etcd reaches the limit set in quota-backend-bytes?

When etcd reaches the limit set by the `quota-backend-bytes` parameter, it switches to "read-only" mode. This means that the etcd database stops accepting new entries but remains available for reading data. To resolve this situation, the following steps need to be taken:

> 1. Changes need to be made in `/etc/kubernetes/manifests/etcd.yaml`, find the line containing `--quota-backend-bytes`. If it does not exist, you will need to add it. For example: `- --quota-backend-bytes=8589934592` - this sets the limit to 8 GB.

Reviewer: ```suggestion
1. Make change to `/etc/kubernetes/manifests/etcd.yaml` — find the line with `--quota-backend-bytes` and edit it. If there is no such line — add, for example: `- --quota-backend-bytes=8589934592` - this sets the limit to 8 GB.
```

> 2. Disarm the active alarm that occurred due to reaching the limit. To do this, execute the following command:

Reviewer: ```suggestion
2. Disarm the active alarm that occurred due to reaching the limit. To do this, execute the command:
```

   ```shell
   kubectl -n kube-system exec -ti $(kubectl -n kube-system get pod -l component=etcd,tier=control-plane -o name | head -n1) -- \
   etcdctl --cacert /etc/kubernetes/pki/etcd/ca.crt \
   --cert /etc/kubernetes/pki/etcd/ca.crt --key /etc/kubernetes/pki/etcd/ca.key \
   --endpoints https://127.0.0.1:2379/ alarm disarm
   ```

> 3. Change the settings [maxDbSize](configuration.html#parameters-apiserver-auditpolicyenabled) in the `control-plane-manager` to that set in the manifest.

Reviewer: ```suggestion
3. Change the [maxDbSize](configuration.html#parameters-apiserver-auditpolicyenabled) parameter in the `control-plane-manager` settings  to match the value specified in the manifest.
```

>    --endpoints https://127.0.0.1:2379/ alarm disarm
>    ```
> 3. Change the settings [maxDbSize](configuration.html#parameters-apiserver-auditpolicyenabled) in the `control-plane-manager` to that set in the manifest.

Reviewer: ```suggestion
   --endpoints https://127.0.0.1:2379/ alarm disarm
   ```

3. Change the settings [maxDbSize](configuration.html#parameters-apiserver-auditpolicyenabled) in the `control-plane-manager` to that set in the manifest.
```
